eureka2-ext/eureka2-eureka1x-rest-api/src/main/java/com/netflix/eureka2/eureka1x/rest/AbstractEureka1xRequestHandler.java

```diff
@@ -0,0 +1,128 @@
+package com.netflix.eureka2.eureka1x.rest;
+
+import javax.ws.rs.core.MediaType;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+import com.netflix.eureka2.eureka1x.rest.codec.CachingEureka1xDataCodec;
+import com.netflix.eureka2.eureka1x.rest.codec.Eureka1xDataCodec;
+import com.netflix.eureka2.eureka1x.rest.codec.Eureka1xDataCodec.EncodingFormat;
+import com.netflix.eureka2.eureka1x.rest.codec.XStreamEureka1xDataCodec;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.handler.codec.http.HttpHeaders.Names;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.reactivex.netty.protocol.http.server.HttpServerRequest;
+import io.reactivex.netty.protocol.http.server.HttpServerResponse;
+import io.reactivex.netty.protocol.http.server.RequestHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import rx.Observable;
+import rx.functions.Func1;
+import rx.functions.Func2;
+
+/**
+ * @author Tomasz Bak
+ */
+public abstract class AbstractEureka1xRequestHandler implements RequestHandler<ByteBuf, ByteBuf> {
+
+    private static final Logger logger = LoggerFactory.getLogger(AbstractEureka1xRequestHandler.class);
+
+    public static final String ROOT_PATH = "/eureka1x/v2";
```

Comment:
Can we make this ROOT_PATH configurable? E.g. for backwards compatibility it does not align with current eureka1 root path.